rllib/offline/estimators/qreg_torch_model.py

```diff
@@ -1,14 +1,15 @@
 from ray.rllib.models.utils import get_initializer
```

Comment:
**TODO**: please add a TODO on top of this file, acknowledging what needs to be done for this model to be correct if your samples do not have the sequential structure. Put it in the documentation of the class.